python/src/aac/plugins/no_ext_for_final/no_ext_for_final_impl.py

```diff
@@ -36,7 +36,7 @@ def no_extension_for_final(
                         ExecutionMessage(
                             f"Cannot resolve unique type for extension {ext.name}.  Found {ext_definition}",
                             MessageLevel.ERROR,
-                            None,  # figure out a better want to handle this...maybe a util function?
+                            definition.source,  # figure out a better want to handle this...maybe a util function?
```

Comment:
```suggestion
                            definition.source, 
```

Not sure the comment needs to remain since something was used to populate the source portion of the ExecutionMessage?

Reply:
Also, using the `definition.source` is not providing anything useful here. It just tells the user that an AaCFile object was the source with some config info. 
`source=AaCFile(uri='<string>', is_user_editable=True, is_loaded_in_context=True)`